### ISSUE TITLE
Cannot find module 'babel-eslint'

### ISSUE THREAD

**Issue by John0x** [dai-shi/es-beautifier#12]
Hey, here is my .eslintrc.json:
````
{
  "root": true,
  "parser": "babel-eslint",
  "env": {
    "browser": true,
    "node": true
  },
  "extends": [
    "standard"
  ],
  // required to lint *.vue files
  "plugins": [
    "html"
  ],
  
  // add your custom rules here
  "rules": {},
  "globals": {}
}
````

The plugin fails once I hit format with the error:
````
Error: Cannot find module 'babel-eslint'
````

**Comment by dai-shi** [comment]
Hi,

es-beautifier doesn't depend on babel-eslint, at the moment.

Let me confirm one thing: do you happen to  enable the `useEslintrc` option? If so, you only need to install babel-eslint on your side; otherwise, I need to look into why it reads user's eslintrc files.

Thanks.

**Comment by John0x** [comment]
@dai-shi hey, yes, I have enabled useEslintrc option and I have installed babel-eslint locally in the project.

**Comment by dai-shi** [comment]
I tried to reproduce your situation with vscode, and see the error `Error: Failed to load plugin html: Cannot find module 'eslint-plugin-html'`. I don't know why it's different from your error.

I have looked into this issue, but so far I wasn't able to find to customize the library paths.

----

The other note is that since you are using standardjs, the rules might be conflicting with those in es-beautifier which is close to eslint-config-airbnb. See: https://github.com/dai-shi/es-beautifier/blob/master/lib/configs/standard.js

What I would suggest is to use es-beautifier just as an eslint plugin, and pick some rules for your .eslintrc.json.
(https://github.com/dai-shi/es-beautifier#usage-eslint-plugin)
And use the eslint extension (https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) if you are using vscode.
